src/Symfony/Component/HttpClient/composer.json

```diff
@@ -24,7 +24,7 @@
     "require": {
         "php": ">=8.4",
         "psr/log": "^1|^2|^3",
-        "symfony/http-client-contracts": "~3.4.4|^3.5.2",
+        "symfony/http-client-contracts": "~3.4.5|^3.5.3",
```

Comment:
is the bump required?

Reply:
Here is a criteria: tests. If they pass without the bump, then the bump is not needed.